cmd/build/helmify/replacements.go

```diff
@@ -79,6 +79,16 @@ var replacements = map[string]string{
     {{- if .Values.service.loadBalancerIP }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
     {{- end }}
+  {{- end }}`,
+	`HELMSUBST_SERVICE_PORTS: ""`: `{{- if .Values.service.ports }}
```

Comment:
Do we want to conditional on `Values.service.ports`? what if the whole thing is empty? should we only conditional on `Values.service.ports.metrics` since we always require `https-webhook-server` and `Values.service.ports.https`